### ISSUE TITLE
NVDA/JAWS should announce the placeholder text provided at edit fields in command palette.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#12212]
Hello, we have been recently testing out our extension for accessibility bugs, and we believe we may have found one directly linked to vscode. The extension we are using is Bridge to Kubernetes.

https://learn.microsoft.com/en-us/visualstudio/bridge/bridge-to-kubernetes-sample

Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Visual Studio Code Version 1.63.2 (user setup)
- OS Version: Windows 11 Version 21H2 (OS Build 22000.376)

Steps to Reproduce:

1. Turn on NVDA/JAWS.
2. Open Visual studio code application and setup Bridge to Kubernetes extension.
3. Download cluster config file and load 'stats-apo' project.
4. Press 'Ctrl+Shift+P' open command palette in View menu.
5. Select 'Bridge to Kubernetes: Configure' and press enter.
6. Navigate using down arrow key to select 'stats-api' in connect to kubernetes (1/4) stage and press enter.
7. Input '3001' in connect to kubernetes service: stats-api (2/4) stage and press enter.
8. Verify whether NVDA/JAWS is announcing the given placeholder text at edit field or not.
